src/main/java/org/gridsuite/modification/server/modifications/AbstractModification.java

```diff
@@ -24,4 +25,8 @@ public void apply(Network network, boolean throwException, ComputationManager co
     public void apply(Network network, Reporter reporter, ApplicationContext context) {
         apply(network, reporter);
     }
+
+    public void control(Network network) throws NetworkModificationException {
```

Comment:
there is already dto::check (before hypothesis creation), so I wanted another name rather than modification::check (before apply).
but why not, I'm ok with it.